src/microbe_stage/organelle_unlocks/UndiscoveredOrganellesTooltip.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Godot;
+
+/// <summary>
+///   The tooltip for undiscovered organelles
+/// </summary>
+public class UndiscoveredOrganellesTooltip : Control, ICustomToolTip
+{
+    [Export]
+    public NodePath? NameLabelPath;
+
+    [Export]
+    public NodePath UnlockTextPath = null!;
+
+#pragma warning disable CA2213
+    private Label? nameLabel;
+    private CustomRichTextLabel? unlockTextLabel;
+#pragma warning restore CA2213
+
+    private string? displayName;
+    private LocalizedStringBuilder? unlockText;
+
+    [Export]
+    public float DisplayDelay { get; set; }
```

Comment:
This should be a field for consistency in Thrive. (so that this can be grouped up with the other export variables).

Reply:
It is required to be a property by the interface this class implements, besides this is done the same way in other custom tooltip classes.

Reply:
Ah, I did not realize. That's the one exception when ExportAttribute is good to use on properties: when it is required by some external factors.